### PR TITLE
Add SPDX headers

### PR DESCRIPTION
**Pull request by AndyScherzinger**

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
@jancborchardt can't you name the source and hence license/author of https://github.com/nextcloud/activity/tree/master/img images?

**Comment by jancborchardt** [comment]
> @jancborchardt can't you name the source and hence license/author of https://github.com/nextcloud/activity/tree/master/img images?

That would be me :D They were contributed under the overall AGPLv3 license.
If possible it would be nice to not include an email address (could be website instead) to prevent spam.

**Comment by AndyScherzinger** [comment]
> If possible it would be nice to not include an email address (could be website instead) to prevent spam.

Yes, doable

File: issue_template.md.license
Comment (nickvergessen) [inline issue_template.md.license:1]: New files in root are problematic for packaging.
Please move the issue template and the license file to `.github`

Comment (AndyScherzinger) [reply issue_template.md.license:1]: Done via https://github.com/nextcloud/activity/pull/1663/commits/0420e4254591156aa1146bba4ff14c2611c10b4d

Comment (AndyScherzinger) [reply issue_template.md.license:1]: > New files in root are problematic for packaging.

We add AUTHORS.md to root with this PR - like on the other apps too.

**Review by nickvergessen — APPROVED** [review]

File: lib/Event/LoadAdditionalScriptsEvent.php [hunk 1]
```diff
@@ -1,23 +1,7 @@
...
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- * GNU Affero General Public License for more details.
- *
- * You should have received a copy of the GNU Affero General Public License, version 3,
- * along with this program. If not, see <http://www.gnu.org/licenses/>
- *
+ * SPDX-FileCopyrightText: 2023 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-only
```
[the remark this answers is not in the document]
Comment (artonge) [reply lib/Event/LoadAdditionalScriptsEvent.php:4]: AGPL-3.0-only ? definitely copy pasted.

[the remark this answers is not in the document]
Comment (artonge) [reply lib/Event/LoadAdditionalScriptsEvent.php:4]: Updated

**Comment by AndyScherzinger** [comment]
All green now @artonge 💯🎉

**Review by jancborchardt — APPROVED** [review]

**Pull request merged**